web/src/components/DatePicker.tsx

```diff
@@ -1,67 +1,30 @@
 import styled from '@emotion/styled'
-import EventIcon from '@mui/icons-material/Event'
-import IconButton from '@mui/material/IconButton'
+import { formHelperTextClasses } from '@mui/material/FormHelperText'
+import { AdapterLuxon } from '@mui/x-date-pickers/AdapterLuxon'
+import { DatePicker } from '@mui/x-date-pickers/DatePicker'
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider'
 import { DateTime } from 'luxon'
 import React, { ReactElement, useEffect, useState } from 'react'
-import DatePicker, { DatePickerProps } from 'react-datepicker'
-import 'react-datepicker/dist/react-datepicker.css'
 import { useTranslation } from 'react-i18next'
 
-import '../styles/DatePickerCalendar.css'
-
-const INPUT_HEIGHT = '56px'
+import useCityContentParams from '../hooks/useCityContentParams'
 
 const DateContainer = styled.div`
   width: fit-content;
   position: relative;
 `
 
-const StyledInputWrapper = styled.div`
-  display: flex;
-`
-
-const StyledIconButton = styled(IconButton)<{ isCalendarOpen: boolean }>`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  position: absolute;
-  right: 16px;
-  align-self: center;
+const StyledError = styled.span`
+  color: ${props => props.theme.colors.invalidInput};
```

Comment:
🔧 Please use the new palette:
```suggestion
  color: ${props => props.theme.palette.error.main};
```

Reply:
`Property 'palette' does not exist on type 'DefaultTheme'.ts`